docs/business/alternate_domains.rst

```diff
@@ -0,0 +1,32 @@
+Alternate Domains
+=================
+
+Read the Docs supports a number of custom domains for your convenience.
+
+Subdomain support
+-----------------
+
+Every project has a subdomain that is available to serve its documentation.
+If you go to ``<organization-slug-project-slug>.readthedocs-hosted.com``,
+it should show you the latest version of documentation.
+
+
+CNAME Support
+-------------
+
+If you have your own domain, you can still host with us.
```

Comment:
To turn a negative sentiment statement into a positive one, this could be something similar to:

`Projects can also be hosted under a custom domain, if you'd prefer to use your own domain name instead of our default hosting domain`